src/workerd/api/crypto/crypto.h

```diff
@@ -207,19 +207,28 @@ class CryptoKey: public jsg::Object {
       KJ_SWITCH_ONEOF(publicExponent) {
         KJ_CASE_ONEOF(array, BigInteger) {
           if (fixPublicExp) {
-            auto expCopy = kj::heapArray<kj::byte>(array.asPtr());
-            jsg::BackingStore expBack = jsg::BackingStore::from(kj::mv(expCopy));
+            // alloc will, by default create a Uint8Array
```

Comment:
If the comment is intended to document what type is hiding behind `auto` then would it not be nicer to just write the type there.

If that's not what is intended, then the reason for the comment is unclear.